partner_deduplicate_by_website/tests/test_crm_deduplicate_by_website.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Tecnativa - Cristina Martin
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+from odoo.tools.safe_eval import safe_eval
+
+
+class TestDeduplicateByWebsite(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner_1 = cls.env['res.partner'].with_context(
+            tracking_disable=True).create({
+                'name': 'Partner 1',
+                'website': 'www.test-deduplicate.com',
+                'email': 'test@deduplicate.com',
+            })
+
+        cls.partner_2 = cls.env['res.partner'].with_context(
+            tracking_disable=True).create({
+                'name': 'Partner 2',
+                'website': 'www.test-deduplicate.com',
+                'email': 'test@deduplicate.com',
```

Comment:
```suggestion
                'email': 'test2@deduplicate.com',
```

Emails shouldn't match, to make sure the deduplication is made by website.